src/components/menu/MenuOpener/MenuOpenedByHover.tsx

```diff
@@ -0,0 +1,54 @@
+// (C) 2007-2018 GoodData Corporation
+import * as React from 'react';
+import { IMenuOpenedBySharedProps } from './MenuOpenedBySharedProps';
+import MenuPosition from '../Position/MenuPosition';
+
+export default class MenuOpenedByHover extends React.Component<IMenuOpenedBySharedProps, {}> {
```

Comment:
I think the empty object for state interface can be omited

Reply:
Removed.